docs/blog/2020-02-18-gazette-2019-recap/index.md

```diff
@@ -0,0 +1,200 @@
+
+### 2.17.0: Move webpack compilation hash into app-data.json
+
+(2019-10-21)
+(https://github.com/gatsbyjs/gatsby/compare/gatsby@2.16.5...gatsby@2.17.0)
+
+Alex Fenton (friend and contributor extraordinaire) moved the webpack compilation hash in Gatsby over to a separate file. This was a step forward for Anthony's effort in 2.9.0, making site uploads incremental and reducing the number of changed files per build.
+
+### 2.18.0: Schema rebuilding
+
+(2019-11-19)
+(https://github.com/gatsbyjs/gatsby/compare/gatsby@2.17.17...gatsby@2.18.0)
+
+Vlad (who recently joined the Gatsby team) killed it in his first month (😱) and added support for automatic Schema rebuilding in the Gatsby CLI. This previously required users to restart the gatsby CLI when changing the schema of their gatsby sites.
+
+There’s a _lot_ more that’s not included here including error message upgrades, several performance improvements, React prerelease tests, an in progress TS migration and more.
+
+## 2020
+
+We started off this year on a solid note with the release of Gatsby Builds, the fastest way to build your Gatsby sites in the cloud. In open source land, we've done plenty already as well.
+
+### Internationalizing Gatsby Docs
+
+Nat resumed her effort of helping internationalize the Gatsby documentation. We want Gatsby to be the most accessible way to build for the web. If you're interested in contributing translations in a language of your choice, go check out the translation contribution guidelines!
+
+### Accessible routing
+
+Madalyn released accessible client side routing in Gatsby in 2.19.8. These improvements enable people relying on screen readers to successfully navigate sites built with Gatsby.
+
+### Build performance
+
+Peter has been working hard on making Gatsby faster. He's already done some improvements which have made a big impact on speeding up Gatsby builds for large sites. On some sites, we've seen hour long builds dropping down to under 5 minutes.
+
+### Jobs API
+
+Ward built the new Gatsby Jobs API which enables Gatsby to better distribute CPU intensive work like image processing and HTML generation to workers on local machines and the Cloud.
+
+## What's next?
+
+You might've noticed a trend in our work last year. Lots of it focussed on foundational improvements to Gatsby unlocking even bigger DX wins and new features that were just _not_ possible before.
+
+This year, we're going to build on those improvements with a focus on _improving_ the Gatsby Developer Experience.
+
+This includes
+
+- making the CLI faster and cleaner
+- adding tools for consuming themes easily
+- making switching data sources for your site even easier
+- better error messaging
+- scaling to hundreds of thousands of pages
+
+We'll tell you more as we get closer to these.
+
+I hope you've enjoyed reading this as much as I've enjoyed writing it.
```

Comment:
Probably leave off the last couple lines -- they read more like an email than a blog post :)

Reply:
Yeah, I felt like that too! 😅